### PR TITLE
Mox

### PR DESCRIPTION
**Pull request by naomijub**

### CODE REVIEW

File: lib/translixir.ex [hunk 1]
```diff
@@ -52,7 +55,7 @@ defmodule Translixir do
   def tx_log!(client, actions) when is_pid(client) do
     url = Client.endpoint(client, :tx_log)
     headers = Client.headers(client)
-    response = HTTPoison.post(url, "#{actions}", headers)
+    response = @http_client.post(url, "#{actions}", headers)
```
Comment (rogerleite) [inline lib/translixir.ex:58]: Out of curiosity, Is there a reason to `actions` be wrapped by `string` in these cases?

File: test/test_helper.exs [hunk 2]
```diff
@@ -1 +1,3 @@
 ExUnit.start()
+Mox.defmock(Translixir.MockHTTPoison, for: HTTPoison.Base)
+HTTPoison.start()
```
Comment (rogerleite) [inline test/test_helper.exs:3]: Probably this is the fix to that previous error that you showed me up, right?

File: test/translixir_happy_path_test.exs [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+                 "crux.db/valid-time": ~U[2020-11-01 02:33:13.371Z],
+                 "crux.tx/tx-id": 0,
+                 "crux.tx/tx-time": ~U[2020-11-01 02:33:13.371Z]
+               },
+               %{
+                 "crux.db/content-hash": %Eden.Tag{
+                   name: "crux/id",
+                   value: "d5c474af4ced822d951d0d2da2d75cf946bca62c"
+                 },
+                 "crux.db/doc": %{"crux.db/id": :jorge, name: "hello"},
```
Comment (rogerleite) [inline test/translixir_happy_path_test.exs:233]: I saw that sometimes has `%Eden.Tag` and other times no. Is there a reason to use Eden.Tag?

**Review by rogerleite — APPROVED** [review]
Good job! 🎯 👍

File: lib/translixir.ex (hunk 1 above)
Comment (naomijub) [reply lib/translixir.ex:58]: Edn format :(

File: test/test_helper.exs (hunk 2 above)
Comment (naomijub) [reply test/test_helper.exs:3]: Yes kkkkk

File: test/translixir_happy_path_test.exs (hunk 3 above)
Comment (naomijub) [reply test/translixir_happy_path_test.exs:233]: This depends on the result of `Eden.decode`, also sometimes for testing reasons I just encode it back

**Pull request merged**